### ISSUE TITLE
Strangle Integration

### ISSUE THREAD

**Issue by LeSnake04** [Heroic-Games-Launcher/HeroicGamesLauncher#2635]
### Problem description

Currently, it's impossible to limit fps in opengl, vulkan and dx12

### Feature description

Add a strangle setting in the UI (maybe replacing the exiting fps limit option).

Strangle is a really useful fps limiting tool that works way more reliable than the MangoHud one and supports pretty much all APIs

### Additional information

Someone probably should make a Strangle flatpak to make the integration easier and so other programs in the flatpak ecosystem can use it.

**Comment by arielj** [comment]
Can't this be handled by the `Wrapper command` configuration we have in game settings > advanced?
![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/188464/cf3dfdc3-8480-488c-9de5-c27a8fef0978)

**Comment by gabriele2000** [comment]
Suggesting a library that hasn't seen a commit since 9 years is crazy... honestly at this point it's just funny, I'm not even mad anymore.
Not trying to insult you @LeSnake04, your suggestion stems from the fact that something essential was removed/made harder to set for the average user, and while I'm not an average user the problem is nonetheless real.

**Comment by arielj** [comment]
> Suggesting a library that hasn't seen a commit since 9 years is crazy... honestly at this point it's just funny, I'm not even mad anymore. Not trying to insult you [@LeSnake04](https://github.com/LeSnake04), your suggestion stems from the fact that something essential was removed/made harder to set for the average user, and while I'm not an average user the problem is nonetheless real.

you are answering to an issue from 2023, it's totally unrelated to the removal of the FPS limit option

**Comment by DeeDeeRanged** [comment]
> Suggesting a library that hasn't seen a commit since 9 years is crazy... honestly at this point it's just funny, I'm not even mad anymore. Not trying to insult you [@LeSnake04](https://github.com/LeSnake04), your suggestion stems from the fact that something essential was removed/made harder to set for the average user, and while I'm not an average user the problem is nonetheless real.

Did you also follow the link to gitlab "gitlab.com/torkel104/libstrangle" you would have seen it is has seen updates more recent.

**Comment by LeSnake04** [comment]
Last time I looked into fps limiters (3-4 years ago) I could only get strangle to work reliably. Also I'm pretty sure the games I launched somewhat recently where I still had that wrapper it still worked. Haven't tested for bugs/lags though. 

If mangohuds limiter works reliable now and that would be a useful option as well (if that isnt present already, haven't used heroic much recently TBH)

PS: I'm also pretty sure strangle worked via the wrappers menu. And I'm pretty sure I used the torkel104 fork.

**Comment by gabriele2000** [comment]
> Did you also follow the link to gitlab "gitlab.com/torkel104/libstrangle" you would have seen it is has seen updates more recent.

I didn't, thanks for the heads-up.

**Comment by gabriele2000** [comment]
> you are answering to an issue from 2023, it's totally unrelated to the removal of the FPS limit option

Dayumn, why is still open then?
Didn't even see the date.

**Comment by arielj** [comment]
> Dayumn, why is still open then? Didn't even see the date.

because there was no answer after my comment to know if it was actually solved or not

and because nobody closed it and it's really hard to follow up with all github issues over the years, there are over 600

**Comment by gabriele2000** [comment]
> because there was no answer after my comment to know if it was actually solved or not
> 
> and because nobody closed it and it's really hard to follow up with all github issues over the years, there are over 600

Right... however I replied to this issue because it was quoted somewhere, I thought it was new, my bad.
